params-2025x.json

```diff
@@ -0,0 +1,1468 @@
+{
+    "materials": {
+        "generic": {
+            "name": "Enterprise PBR Shading Model",
+            "preview": "img/subtypes_2025x_basic.png",
+            "parameters": {
+                "albedo": {
+                    "name": "Albedo",
+                    "type": "color",
+                    "max": [1, 1, 1],
+                    "default": [1, 1, 1],
+                    "uiOrder": 0,
+                    "group": "Base"
+                },
+                "metallic": {
+                    "name": "Metallic",
+                    "type": "float",
+                    "min": 0,
+                    "max": 1,
+                    "default": 0,
+                    "uiOrder": 1,
+                    "group": "Base"
+                },
+                "roughness": {
+                    "name": "Roughness",
+                    "type": "float",
+                    "min": 0,
+                    "max": 1,
+                    "default": 0,
+                    "uiOrder": 2,
+                    "group": "Base"
+                },
+                "anisotropy": {
+                    "name": "Anisotropy",
+                    "type": "float",
+                    "min": 0,
+                    "max": 1,
+                    "default": 0,
+                    "uiOrder": 3,
+                    "group": "Base"
+                },
+                "anisotropyRotation": {
+                    "name": "Anisotropy Rotation",
+                    "type": "float",
+                    "min": 0,
+                    "max": 1,
+                    "default": 0,
+                    "uiOrder": 4,
+                    "group": "Base"
+                },
+                "normal": {
+                    "name": "Normal",
+                    "type": "normal",
+                    "default": [0, 0, 1],
+                    "uiOrder": 5,
+                    "group": "Base"
+                },
+                "displacement": {
+                    "name": "Displacement",
+                    "type": "vector",
+                    "default": [0, 0, 0],
+                    "uiOrder": 6,
+                    "group": "Base"
+                },
+                "translucency": {
+                    "name": "Translucency",
+                    "type": "float",
+                    "min": 0,
+                    "max": 1,
+                    "default": 0,
+                    "uiOrder": 7,
+                    "group": "Base"
+                },
+                "translucencyColor": {
+                    "name": "Translucency Color",
+                    "type": "color",
+                    "max": [1, 1, 1],
+                    "default": {
+                        "$ref": "#/materials/generic/parameters/albedo/default"
```

Comment:
This is not part of the params format spec (see Sec. 6.4) and not supported by the JSON viewer (param-viewer.html). Is it supported by the applications consuming the JSON file? If yes, keep it, but add it to Sec. 6.4 and adapt the JSON viewer.